### PR TITLE
aws-operator: extract project.BundleVersion()

### PR DESCRIPTION
**Pull request by kopiczko**
Towards https://github.com/giantswarm/giantswarm/issues/7345

### CODE REVIEW

File: pkg/project/project.go [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
-	version            = "n/a"
+	bundleVersion        = "5.5.0"
+	description          = "The aws-operator handles Kubernetes clusters running on a Kubernetes cluster inside of AWS."
+	gitSHA               = "n/a"
+	name          string = "aws-operator"
+	source        string = "https://github.com/giantswarm/aws-operator"
+	version              = "n/a"
 )
 
+func BundleVersion() string {
```
Comment (kopiczko) [inline pkg/project/project.go]: This is taken from `master` branch. I'm not happy with the name. But we will get rid of version bundle concept later.

File: pkg/project/project.go [hunk 2]
```diff
@@ -1,13 +1,18 @@
 package project
 
 var (
-	description        = "The aws-operator handles Kubernetes clusters running on a Kubernetes cluster inside of AWS."
-	gitSHA             = "n/a"
-	name        string = "aws-operator"
-	source      string = "https://github.com/giantswarm/aws-operator"
-	version            = "n/a"
+	bundleVersion        = "5.5.0"
```
Comment (MarcelMue) [inline pkg/project/project.go]: So after this one gets tagged, the next commit should increase this number, right?

Comment (kopiczko) [reply pkg/project/project.go]: Yes for now. We are working towards removing this const.

Comment (kopiczko) [reply pkg/project/project.go]: https://github.com/giantswarm/giantswarm/blob/master/processes/release-an-operator.md#release-a-new-operator-version-to-all-installations

**Pull request merged**